braph2genesis/graphs/_MultilayerBUT.gen.m

```diff
@@ -2,9 +2,13 @@
 MultilayerBUT < MultilayerWU (g, multilayer binary undirected with fixed threaholds graph) is a multilayer binary undirected with fixed threaholds graph.
 
 %%% ¡description!
-In a multilayer binary undirected with fixed threaholds (BUT) graph, layers have different number 
- of nodes with within-layer binary undirected (BU) edged matrix  at different thresholds.
-There are connections between layers connecting the corresponding nodes.
+In a multilayer binary undirected with fixed thresholds (BUT) graph, layers 
+consist of binary undirected (BU) multilayer graphs derived from the same
+weighted supra-connectivity matrices binarized at different thresholds. Layers 
+within the binary undirected (BU) multilayer graphs could have different number of nodes
+with within-layer binary undirected edges. Edges can be either 0 (absence of connection)
+or 1 (existence of connection).
+All node connections are allowed between layers.
```

Comment:
In a multilayer binary undirected with fixed thresholds (BUT) graph, the layers are those of binary undirected (BU) multiplex graphs derived from the same weighted supra-adjacency matrix binarized at different thresholds. The supra-adjacency matrix has a number of partitions equal to the number of thresholds. Layers 
within the binary undirected (BU) multilayer graphs could have different number of nodes
with within-layer binary undirected edges. Edges can be either 0 (absence of connection)
or 1 (existence of connection).
All node connections are allowed between layers.